### PR TITLE
User guide and what's new: linguistic fixes, add missing anchoir tag

### PR DESCRIPTION
**Pull request by josephsl**
### Link to issue number:
None

### Summary of the issue:
Add missing anchor in user guide so proper differences delta can be calculated for translators, linguistic fixes for what's new, add one or more things in features list in user guide.

### Description of how this pull request fixes the issue:
For a while now, translators have been complaining about wrong differences file for user guide. It turns out a missing left bracket in one of the anchors was the cause.
Also, added a note about touchscreens in features list.
Linguistic fixes for what's new.

### Testing performed:
Compiled with .t2t to .html.

### Known issues with pull request:
None

### Change log entry:
None

### CODE REVIEW

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -8,17 +8,17 @@ What's New in NVDA
 == Changes ==
 - The HIMS Braille Sense/Braille EDGE/Smart Beetle and Hims Sync Braille display drivers have been replaced by one driver. The new driver will automatically be activated for former syncBraille driver users. (#7459) 
  - Some keys , notably scroll keys, have been reassigned to follow the conventions used by Hims products. Consult the user guide for more details.
-- When typing with the on-screen keyboard via touch interaction, by default  you now need to double tap each key the same way you would activate any other control.
- - To use the existing "touch typing" mode where simply lifting your finger off the key is enough to activate it, Enable this option in the new Touch Interaction settings dialog found in the Preferences menu. (#7309)
+- When typing with the on-screen keyboard via touch interaction, by default  you now need to double tap each key the same way you would activate any other control. (#7309)
```
Comment (feerrenrut) [inline user_docs/en/changes.t2t]: Since you are here, there is a double space between "default" and "you"

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -8,17 +8,17 @@ What's New in NVDA
...
-- When typing with the on-screen keyboard via touch interaction, by default  you now need to double tap each key the same way you would activate any other control.
- - To use the existing "touch typing" mode where simply lifting your finger off the key is enough to activate it, Enable this option in the new Touch Interaction settings dialog found in the Preferences menu. (#7309)
+- When typing with the on-screen keyboard via touch interaction, by default  you now need to double tap each key the same way you would activate any other control. (#7309)
+ - To use the existing "touch typing" mode where simply lifting your finger off the key is enough to activate it, Enable this option in the new Touch Interaction settings dialog found in the Preferences menu.
 
 
 == Bug Fixes ==
-- Browseable messages such as showing  current formatting when pressing NVDA+f twice quickly, no longer fails when NVDA is installed on a path with non-ASCII characters. (#7474)
+- Browseable messages such as showing  current formatting when pressing NVDA+f twice quickly no longer fails when NVDA is installed on a path with non-ASCII characters. (#7474)
```
Comment (feerrenrut) [inline user_docs/en/changes.t2t]: huh... also a double space between "showing" and "current"

**Review by feerrenrut — APPROVED** [review]

**Comment by feerrenrut** [comment]
Give me a yell when you are happy with this, and have resolved conflicts and I will merge it in.

**Comment by josephsl** [comment]
Hi, merged in latest master, fixing conflicts and correct spelling for PowerPoint. I’m ready for this to go to master directly. Thanks.

**Comment by bdorer** [comment]
Hi, I didn't read all spelling fixes, but I found the following typos in 
the userguide yesterday:
==== Beep if Typing Lowercase Letters when Caps Lock is On 
====[VoiceSettingsBeepLowercase] should be keyboard
==== Speak Command Keys ====[VoiceSettingsSpeakCommandKeys] should be 
keyboardSetting
==== Report Object descriptions 
====ObjectPresentationReportDescriptions] missing [ as the open anchor tag
+++ Browse Mode Settings (NVDA+control+b) +++BrowseModeSettings] same as 
above
==== Configuring reporting of fields such as links and headings ====: 
has no Anchortag as all other Options is this wanted?

If you've fixed them allready in this pr feel free to ignore this message.

Am 02.01.2018 um 07:47 schrieb Reef Turner:

**Comment by josephsl** [comment]
Hi, if nobody wants to do it, I’ll take a look at these, as Tom should have gone over them through his work on anchors. For now, I recommend a separate issue on documentation problems and let someone new take a look at it. Thanks.

**Comment by bdorer** [comment]
Hi, I could do it myself, but I'm not familliar working with pull 
requests. I saw some typos I mentioned are fixed allready.

Am 02.01.2018 um 12:07 schrieb Joseph Lee:

**Pull request merged**